stan/math/prim/prob/multi_normal_lpdf.hpp

```diff
@@ -102,6 +103,24 @@ return_type_t<T_y, T_loc, T_covar> multi_normal_lpdf(const T_y& y,
   return lp;
 }
 
+template <bool propto, typename T_y, typename T_loc, typename T_covar,
+          require_all_not_vector_t<T_y, T_loc>* = nullptr>
+return_type_t<T_y, T_loc, T_covar> multi_normal_lpdf(const T_y& y,
+                                                     const T_loc& mu,
+                                                     const T_covar& Sigma) {
+  using lp_type = return_type_t<T_y, T_loc, T_covar>;
+
+  const size_t N = y.rows();
+  lp_type lp(0.0);
```

Comment:
Could you take away the `using lp_type...` statement on 111 and declare `lp` as that type? It's easier to read when that's together.